Backpack/Calendar/Classes/BPKCalendarStickyHeader.m

```diff
@@ -86,8 +88,9 @@ - (void)layoutSubviews {
     [NSLayoutConstraint activateConstraints:@[
         [self.titleLabel.centerYAnchor constraintEqualToAnchor:self.contentView.centerYAnchor],
         [self.titleLabel.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor],
-
-        [self.selectMonthButton.centerYAnchor constraintEqualToAnchor:self.titleLabel.lastBaselineAnchor constant:BPKBaselineOffset],
+        [self.titleLabel.heightAnchor constraintEqualToConstant:BPKSpacingLg * 2],
+        [self.titleLabel.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor],
```

Comment:
Make the label full width for easier reaching when exploring by touch.
This does not interfere with the whole month button